packages/copy-widget-code/src/main.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import {appFactory, actionEmitter, externalEvents, notification} from 'tocco-app-extensions'
+import {reducer as reducerUtil} from 'tocco-util'
+
+import CopyWidgetCode from './components/CopyWidgetCodeContainer'
+import reducers, {sagas} from './modules/reducers'
+
+const packageName = 'copy-widget-code'
```

Comment:
I'm currently working on the edit action for the widgets and I called the package `widget-config-edit-action` (like a mix between `input-edit` and `user-qr-action` which already exist).

I think we should use the same naming pattern for our new actions, so either:

copy-widget-code(-action)
edit- widget-config(-action)

or:

widget-code-copy(-action)
widget-config-edit(-action)

which one of the patterns should we use? and with or without the -action suffix?